inference/python/infbench/cutlass.py

```diff
@@ -56,6 +154,7 @@ def __init__(self, modelArg):
         KaaS model. If a path is passed, it should be a directory containing:
         name.cubin, name_meta.yaml, and name_model.yaml (where name is the
         name of the directory)."""
+        print(modelArg)
```

Comment:
clean up old debugging code. A trick I use is to throw #XXX next to stuff I don't want to commit. The linter will complain and you can go clean it up. e.g.:

    print("foobar") #XXX